staging/nvidia/charts/nvidia-device-plugin/values.yaml

```diff
@@ -0,0 +1,60 @@
+# Default values for nvidia-device-plugin.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 1
+
+image:
+  repository: nvidia/k8s-device-plugin
+  tag: 1.0.0-beta
+  pullPolicy: IfNotPresent
+
+imagePullSecrets: []
+nameOverride: ""
+fullnameOverride: ""
+
+service:
+  type: ClusterIP
+  port: 80
+
+ingress:
+  enabled: false
+  annotations: {}
+    # kubernetes.io/ingress.class: nginx
+    # kubernetes.io/tls-acme: "true"
+  hosts:
+    - host: chart-example.local
+      paths: []
+
+  tls: []
+  #  - secretName: chart-example-tls
+  #    hosts:
+  #      - chart-example.local
+
+resources: {}
+  # We usually recommend not to specify default resources and to leave this as a conscious
+  # choice for the user. This also increases chances charts run on environments with little
+  # resources, such as Minikube. If you do want to specify resources, uncomment the following
+  # lines, adjust them as necessary, and remove the curly braces after 'resources:'.
+  # limits:
```

Comment:
we should set some default resources

Reply:
so we can adjust them in the next MWT

Reply:
I was following examples in this repo. Were they misbehaving previously? Or we want to make this enforcement starting from now?

- Daemonsets like `awscsiprovisioner`, `localvolumeprovisioner`.
- Other deployments that haven't set up a default resource yet - `dex-k8s-authenticator`, `kube-oidc-proxy`, `prometheus-node-exporter` and `traefik-forward-auth`

I am happy to figure out what is the appropriate default resources for the Nvidia Chart. Not sure if we can address it together with other charts

Reply:
@Gilbert88 hold on! is this a critical daemon ? you are comparing it with the awscsiprovisioner, if so you are missing some properties such as `priorityClassName: system-node-critical`

`prometheus` gets his resource definitions in kubeaddons-configs. https://github.com/mesosphere/kubeaddons-configs/blob/master/templates/prometheusadapter.yaml#L32 https://github.com/mesosphere/kubeaddons-configs/blob/master/templates/prometheus.yaml#L182
https://github.com/mesosphere/kubeaddons-configs/blob/master/templates/prometheus.yaml#L193

`kube-oidc-proxy` has been removed....
`dex-k8s-authenticator` https://github.com/mesosphere/kubeaddons-configs/blob/master/templates/dex-k8s-authenticator.yaml#L54


`traefik-forward-auth` https://github.com/mesosphere/kubeaddons-configs/blob/master/templates/traefik-forward-auth.yaml#L23

Reply:
Please, add some estimations in the kubeaddons-configs PR. Don't set them here if you prefer.

Reply:
Spent a couple hours to test out different combinations of default resources. I believe the value in https://github.com/mesosphere/kubeaddons-configs/pull/212 will be close to what we expect.